kodilla-basic-tests/src/main/java/com/kodilla/bank/homework/CashMachine.java

```diff
@@ -0,0 +1,84 @@
+package com.kodilla.bank.homework;
+
+public class CashMachine {
+    private int[] transactions;
+    private int size;
+
+    public CashMachine() {
```

Comment:
public CashMachine(int[] transactions) {
        this.transactions = transactions;
        this.size = transactions.length;
    }